packages/microsoft_sqlserver/manifest.yml

```
@@ -1,33 +1,74 @@
 format_version: 1.0.0
 name: microsoft_sqlserver
 title: "Microsoft SQL Server"
-version: 1.0.0
+version: 1.0.1
```

As you are changing the kibana version constraint, please increase the version of the package to `1.1.0`, so fixes can be still be made for older versions as 1.0.X.
```suggestion
version: 1.1.0
```

---

sounds good.